tritium-metrics-jvm/src/main/metrics/metrics.yml

```diff
@@ -37,3 +37,10 @@ namespaces:
       attribute.uptime:
         type: gauge
         docs: Provides the Java virtual machine uptime value in milliseconds.
+        tags:
+        - javaSpecificationVersion # 11
+        - javaVersion # 11.0.3
+        - javaVersionDate # 11.0.3
+        - javaRuntimeVersion # 11.0.3+7-LTS
+        - javaVendorVersion # Zulu11.31+11-CA
+        - javaVmVendor # Azul Systems, Inc.
```

Comment:
Might as well list examples in the docs field so everyone gets the benefits.

Reply:
Sure thing - I wonder if it would actually make sense to allow documentation on tags themselves? currently it's just a list<string>

Reply:
I had considered that but ultimately decided against it because I've found the conjure format with multiple value structures confusing. I think we need docs for tags infrequently enough that its worth the simplicity to include documentation in the metric docs field.
I may have gotten it wrong though ¯\_(ツ)_/¯ What do you think?

Reply:
yeah I think it's probably infrequent enough that there's not really any point. If we weren't committed to a conjure def here I think we could probably do some kind of fun yaml abuse like:

```yml
tags:
- javaRuntimeVersion : docs for this tag
- javaVendorVrrsion
- etc
```